src/Raven.Server/ServerWide/DebugInfoPackageUtils.cs

```diff
@@ -75,5 +75,23 @@ await using (var sw = new StreamWriter(entryStream))
                 await sw.FlushAsync();
             }
         }
+
+        public static async Task WriteExtraDebugInfoAsZipEntryAsync(Dictionary<string, TimeSpan> debugInfoTimeSpans, ZipArchive archive, string databaseName)
```

Comment:
I don't like the `Extra`.
It doesn't tell me about the content.
Maybe something like WriteDebugInfoMetadataAsZipEntryAsync for a generic manner
Or WriteDebugInfoTimesAsZipEntryAsync for specific manner

Reply:
done